src/chigame/games/forms.py

```diff
@@ -16,7 +16,5 @@ class Meta:
         }
         widgets = {
             "name": forms.TextInput,
-            "category": forms.CheckboxSelectMultiple,
```

Comment:
Why do we want to remove category and mechanics checkboxes from forms?

Reply:
The number of checkboxes becomes infeasible.

<img width="373" alt="Screenshot 2023-11-15 at 5 03 11 PM" src="https://github.com/uchicago-cs/chigame/assets/69015620/5b52487e-16c5-4b37-9e01-560fcbad87a0">